modules/timeutil/datetime.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package timeutil
+
+import (
+	"fmt"
+	"html/template"
+)
+
+// DateTime renders an absolute time HTML given a time as a string
+func DateTime(format, datetime, fallback string) template.HTML {
+	switch format {
+	case "short":
+		return template.HTML(fmt.Sprintf(`<relative-time format="datetime" year="numeric" month="short" day="numeric" weekday="" datetime="%s">%s</relative-time>`, datetime, fallback))
+	case "long":
+		return template.HTML(fmt.Sprintf(`<relative-time format="datetime" year="numeric" month="long" day="numeric" weekday="" datetime="%s">%s</relative-time>`, datetime, fallback))
+	case "full":
+		return template.HTML(fmt.Sprintf(`<relative-time format="datetime" weekday="" year="numeric" month="short" day="numeric" hour="numeric" minute="numeric" second="numeric" datetime="%s">%s</relative-time>`, datetime, fallback))
+	}
```

Comment:
Since `datetime` and `fallback` are all passed in as string, they need to be escaped before being used in HTML.

Reply:
What's the recommended way to do that?

Reply:
```
datetimeEscaped := html.EscapeStirng(datetime)
fallbackEscaped := html.EscapeStirng(fallback)
Sprintf("... %s ... %s ...", datetimeEscaped, fallbackEscaped)
```